htmresearch/support/lateral_pooler/datasets.py

```diff
@@ -0,0 +1,169 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2017, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+import numpy as np
+from scalar_encoder import ScalarEncoder 
+from scipy.io import loadmat
+from sklearn.datasets import fetch_mldata
```

Comment:
This code won't run by default if the libraries aren't in the projects requirements.txt file. Fine to leave them out if the lateral pooler is still experimental but you might want to indicate that in the module doc string in that file.